.github/workflows/release.yml

```diff
@@ -101,20 +101,20 @@ jobs:
       matrix:
         os: ['macOS', 'Windows', 'Linux']
         py-version: ['3.7', '3.8', '3.9', '3.10']
-        tf-version: ['2.10.0']
+        tf-version: ['2.9.3', '2.10.1', '2.11.0']
```

Comment:
We only publish wheels that were built with the most recent TF version.

Reply:
Yes we had some drift with https://github.com/tensorflow/addons/blob/master/tools/update_release_version.sh.
I think that we could fix that utility that I've contributed as it is still useful to quickly prepare a release version PR for the "repetitive" tasks.